R/get_axes.R

```diff
@@ -0,0 +1,28 @@
+#' Get plot Axes
+#'
+#' These functions extract just the axes from a ggplot. `get_y_axis()` pulls
+#' the y-axis, while `get_x_axis()` pulls the x-axis.
+#'
+#' @param plot A ggplot or gtable.
+#' @param side Which side of the plot is the axis on?
```

Comment:
Could you add a sentence or two stating what the possible options for `side` are? I'm also wondering whether they should be spelled out, since the rest of ggplot2 uses "top", "bottom", etc. Does `match.arg()` do partial matching, so that both "top" and "t" could work?